zulip_bots/zulip_bots/bots/tictactoe/tictactoe.py

```diff
@@ -293,22 +306,31 @@ def handle_message(self, message, bot_handler):
             storage.put(original_sender, user_board)
         user_game = TicTacToeGame(user_board) if user_board else None
 
+        valid_command = TicTacToeGame.check_validity(user_game, TicTacToeGame.sanitize_move(user_game, command))
+        valid_number = TicTacToeGame.check_numbers(user_game, TicTacToeGame.sanitize_move(user_game, command))
```

Comment:
Can you name these `is_valid_command` and `is_valid_number`? The `is_` is a naming convention that indicates a boolean.